core/partitioning/partitioning.cpp

```diff
@@ -31,11 +31,15 @@ bool containNonTensorOutputs(torch::jit::Node* n) {
 }
 
 bool isModifyingNodes(torch::jit::Node* node, torch::jit::Value* val) {
-  const auto& schema = node->schema();
+  const torch::jit::FunctionSchema* schema = node->maybeSchema();
+  if (!schema) {
+    return false;
+  }
   for (size_t i = 0; i < node->inputs().size(); ++i) {
     if (node->inputs()[i] == val) {
-      const at::AliasInfo* formal = schema.arguments()[i].alias_info();
+      const at::AliasInfo* formal = schema->arguments()[i].alias_info();
       if (formal && formal->isWrite()) {
+        LOG_GRAPH("<Whatever is doing the modifying> Is modifying node " << util::node_info(node));
```

Comment:
😄 I didn't mean put "<Whatever is doing the modifying>" verbatim, just what ever component is running this. not sure if its part of shape analysis or some other phase of partitioning

Reply:
lol. Ok let me change it.